### PR TITLE
use html5 upload and revert to flash if necessary

### PR DESCRIPTION
**Pull request by andrewroth**

### CODE REVIEW

File: app/controllers/fe/concerns/answer_pages_controller_concern.rb [hunk 1]
```diff
@@ -60,16 +61,17 @@ def update
   end
 
   def save_file
-    params.permit(:Filedata)
+    params.permit(:Filefile)
```
Comment (knutsenm) [inline app/controllers/fe/concerns/answer_pages_controller_concern.rb]: Is "Filefile" a typo?

File: app/views/fe/questions/fe/_attachment_field.html.erb [hunk 2]
```diff
@@ -1,22 +1,25 @@
+<% element_id = attachment_field.id; base_element = "#element_#{attachment_field.id}" %>
```
Comment (knutsenm) [inline app/views/fe/questions/fe/_attachment_field.html.erb]: or `base_element = "#element_#{element_id}"`

File: app/controllers/fe/concerns/answer_pages_controller_concern.rb (hunk 1 above)
Comment (andrewroth) [reply app/controllers/fe/concerns/answer_pages_controller_concern.rb]: Yes, thanks

**Comment by twinge** [comment]
I'm assuming that `jquery.html5_upload.js` was dropped in from somewhere else, not something @andrewroth wrote himself.

**Comment by andrewroth** [comment]
Yes, comes from https://github.com/mihaild/jquery-html5-upload; I can put that as a comment in the file..

**Pull request merged**